extension-localization/src/main/java/com/mapbox/maps/extension/localization/StyleInterfaceExtension.kt

```diff
@@ -5,7 +5,10 @@ import java.util.*
 
 /**
  * Extension function to localize style labels
+ *
+ * @param locale the locale that applied for localization
+ * @param layerIds the id of layers that will localize on, default is null which means will localize all the feasible layers.
  */
-fun StyleInterface.localizeLabels(locale: Locale) {
-  setMapLanguage(locale, this)
+fun StyleInterface.localizeLabels(locale: Locale, layerIds: List<String>? = null) {
```

Comment:
Perhaps use `emptyList()` as default parameter and make `layerIds` not nullable then? imho it will work the same but code will be cleaner

Reply:
As shown in this [line](https://github.com/mapbox/mapbox-maps-android/pull/461/files#diff-4d722ef66fccbb18db39c8a9f28f6d79472a40ca68110bf356983e5888eb649bR29) a nullable list can make the code more cleaner.

Reply:
Wouldn't `layerIds.contains(layer.id)` work the same here? 🤔

Reply:
I see now, I guess not.
My only concern - will the code work OK if user would pass `emptyList()` explicitly?

Reply:
This list contains the layers they want to localize, an empty list means no layers will be localized and the filter will filter out all the layers.

Reply:
So having an empty list is equivalent to passing null. In that case I'd make this parameter not nullable if it acts the same as null and pass empty list as default.

Reply:
If I understand correctly, passing empty list will result in no localization at any layer at all, please correct me if I'm wrong @Chaoba

Reply:
I see that iOS is also using null by default, ok let's leave it this way to be aligned. Although passing empty list basically meaning that function will not work at all seems pretty useless to me.

Reply:
Passing null will localize all feasible layers, which is the default action, while an empty list will not localize any layer.